_unittest/test_41_3dlayout_modeler.py

```diff
@@ -287,6 +287,10 @@ def test_12_create_line(self):
             "Bottom", [[0, 0], [10, 30], [20, 30]], lw=1, name="line1", net_name="VCC"
         )
         assert line.name == "line1"
+        assert isinstance(line.center_line, dict)
+        line.center_line = {"Pt0": [1, "0mm"]}
```

Comment:
@maxcapodi78 Shouldn't we provide list of `Points3dLayout` objects instead of a dictionary containing the name of the point as a key.

Reply:
I made it as a dict since user may want to change just 1 point value and in this way he should not need to have the full list provided any time

Reply:
Yes, that could avoid the users to have to do a `for loop` to get the point they are interested to